src/quotes/quotes.js

```diff
@@ -0,0 +1,163 @@
+import { FRONT_END_ROLE, BACK_END_ROLE, DEV_OPS_ROLE } from '../constants/roles.js';
+import { BRAG, CONFESS } from '../constants/roleActions.js';
+
+const SUCCESS = BRAG;
+const FAIL = CONFESS;
+export const PAST = 'past';
+export const FUTURE = 'future';
+const DEVOPS = DEV_OPS_ROLE;
+const BACKEND = BACK_END_ROLE;
+const FRONTEND = FRONT_END_ROLE;
+export const PREFIX = 'prefix';
+
+// Still consider this experimental
+// ORDER HERE MATTERS, ALWAYS ADD AT THE END
+const PREFIX_LEAF = {
```

Comment:
Why `LEAF` sufix?

Reply:
Cause its a leaf :)

Reply:
For sake of sanity of mere humans maybe just call it 'prefixes'?